src/Tools/MicroBuild/post-build.ps1

```diff
@@ -0,0 +1,79 @@
+# Run our post build steps.  This occurs after all of our shipping binaries are built and fully 
+# signed.  The steps we need to execute are:
+#
+#   1. Run GitLink on the PDBs so we have proper source linking.
+#   2. Setup the <Configuration>\Index directory for simple / efficient indexing in official builds
+
+
+param([string]$config = "Release")
+Set-StrictMode -version 2.0
+$ErrorActionPreference="Stop"
+
+# Get the path to all of the shipping EXE / DLL files.
```

Comment:
This should also include unit tests and other stuff too; we do want those to work.